dolphin/processor/config.py

```diff
@@ -288,8 +289,37 @@ def get_kwargs_likelihood(self):
                     prior_param.extend(i)
                     kwargs_likelihood['prior_ps'].append(prior_param)
 
+        if 'lens_light_option' in self.settings and \
+                'prior_lens_light_ellip' in self.settings['lens_light_option']:
```

Comment:
Use full words in variable names, e.g., `prior_lens_light_ellipticity`. But, I would suggest for it to be more descriptive, so I recommend changing it to `constrain_position_angle_from_lens_light` and put it to under `lens_option`. Because, the prior is more on the lens mass, as the light PA is constrained from the imaging already.